src/c++/vdo/tests/RecoveryJournal_t2.c

```diff
@@ -305,6 +305,38 @@ static BlockPattern partialHeaderPattern[JOURNAL_BLOCKS] = {
   { 11, 15, GOOD_COUNT, USE_NONCE, FULL_BLOCK,  true,  normalSectors     },
 };
 
+/**
+ * A non-wrapped journal that has been corrupted by the old header
+ * version being written in block 4. The reap head is 1 and the
+ * highest sequence number is 5.
+ * 
```

Comment:
Blank line in the comment should be removed.

Reply:
Fixed.